pycode/memilio-epidata/memilio/epidata/modifyDataframeSeries.py

```diff
@@ -326,58 +326,116 @@ def create_intervals_mapping(from_lower_bounds, to_lower_bounds):
 
 
 def fit_age_group_intervals(
-        from_age_groups, population, to_age_groups, max_age=100):
+        df_age_in, age_out, df_population=None, max_age=100):
```

Comment:
We should pay attention to max/min. I used 0-99 since this gives exactly 100 parts...

Reply:
see above